README.md

```diff
@@ -199,3 +199,8 @@ To implement this into your markup, we suggest adding this to your `layouts.defa
 ```
 
 There is a `defaultLanguage` config variable that you are free to customise to suit your application. This package comes with `en-GB` set as the default language.
+
+
+## Access Control
```

Comment:
Could do with some additional docs please. E.g. overload `authoriseToView` or use `accessContent` in the Policy to customise the access control functionality.